WTWSMS/common/religions/00_religions.txt

```diff
@@ -4956,15 +4997,16 @@ indian_group = {
 			THE_VINAYAKAS
 		}
 		
-		autocephalacy = no
+		autocephaly = yes
```

Comment:
Yes, we agreed on that, but the issue is that all pentarchies have to redone (which we agree on too)  but I want Enlil tro look at it before.